live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/model/EventType.java

```diff
@@ -27,5 +27,18 @@
 
 public enum EventType
 {
-    CREATE, UPDATE, DELETE;
+    CREATE_OR_UPDATE("createOrUpdate"), DELETE("delete");
+
+    private final String value;
+
+    EventType(String value)
+    {
+        this.value = value;
+    }
+
+    @Override
+    public String toString()
```

Comment:
I think it will be better crating and using `@Getter` on `value` then overriding `toString()`